docs/documentation/development/bindings/xml-reference.md

```diff
@@ -95,6 +97,7 @@ The following HTML tags are allowed -: ```<b>, <br>, <em>, <h1>, <h2>, <h3>, <h4
   <tr><td>option</td><td>The element definition of a static selection list (optional).</td></tr>
   <tr><td>option.value</td><td>The value of the selection list element.</td></tr>
   <tr><td>multipleLimit</td><td>If multiple is true, sets the maximum number of options that can be selected (optional).</td></tr>
+  <tr><td>unitLabel</td><td>The unit label represents a human readable natural language unit as iterations, runs, etc. If the unit attribute is set then the unit label attribute will be ignored. The unit label must only be set if the type of the parameter is either integer or decimal (optional).</td></tr>
```

Comment:
> then the unit label attribute will be ignored

-> "is ignored"

So this means we do not have an exception when parsing the XML, right?